var/spack/repos/builtin/packages/py-pyrr/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyrr(PythonPackage):
+    """3D mathematical functions using NumPy."""
+
+    homepage = "https://github.com/adamlwgriffiths/Pyrr"
+    pypi     = "pyrr/pyrr-0.10.3.tar.gz"
+    maintainers = ['JeromeDuboisPro']
+    license = "BSD-2-Clause"
```

Comment:
This variable isn't used by Spack as far as I know, can probably remove this.

Reply:
I guess you are refering to the license variable.
During linux checks, I had a failure due to header or license being wrong.
I found a "license=" in some other py-* package, so I tried this. Turned out it was the copyright year range "-2020" -> "-2022" which fixed this failure.
I'll remove the license variable to resolve this item.
Thanks a lot for all the appropriate comments and suggestions 😊.